pxi.py

```diff
@@ -39,6 +39,10 @@ class PXI:
 
     def __init__(self, address: Tuple[str, int]):
         self.logger = logging.getLogger(str(self.__class__))
+        self.logger.setLevel(logging.DEBUG)
+        fh = logging.FileHandler('spam.log')
```

Comment:
I added this to be able to look at the whole history of the log. I don't think it's the best way to do it, rather it's a hacky first pass.

I'm happy to keep it here for now but we should implement something more robust down the line.